mods/ra/sequences/structures.yaml

```diff
@@ -143,8 +138,6 @@ apwr:
 	make: apwrmake
 		Start: 0
 		Length: *
-	dead: apwrdead
-		Start: 0
```

Comment:
This does fix the husks, which removes some of the pressure from removing them.  If there's any controversy over this then l could port the TD fixes and keep them.